_includes/v21.2/prod-deployment/healthy-storage-capacity.md

```diff
@@ -0,0 +1 @@
+**Expected values for a healthy cluster**: Used capacity should be no more than 80% of the total capacity.
```

Comment:
I am pretty sure this 80% is an arbitrary value I picked up through previous threads - unfortunately can't find the exact source now. Do we have more precise guidance? Also, this differs from our Prometheus alert that is preconfigured to alert at less than 15% available capacity.

Reply:
10-15% would be critical. 20% a warning. Also see: https://github.com/cockroachlabs/cockroachdb-runbook-template/blob/main/monitoring-alerts/alert-node-storage-capacity.md